packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -84,6 +84,9 @@ function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
   );
 
   if (devices.length === 0) {
+    return logger.error('No Mac or iOS devices connected.'); // which should never happen when running on Mac
```

Comment:
can we revert it then?

Reply:
Remove this `if`?

Reply:
yes, according to the comment this should never happen, at least on Xcode 11

Reply:
Sure!